content/neuvector-prime/aws/contents.lr

```diff
@@ -0,0 +1,164 @@
+title: NeuVector Prime in AWS: Usage Instructions
+---
+theme: suse
+---
+body:
+
+## Prerequisites
+
+1. These instructions assume you have an EKS cluster installed and that you have access to that cluster via a `kubeconfig`.
+2. Installation requires you have the following tools available and properly configured to access your AWS account, and your EKS cluster:
+* `aws`
+* `curl`
+* `eksctl`
+* `helm` (v3 or greater)
+
+## Preparing your cluster
+
+### OIDC provider
+
+Your EKS cluster is required to have an OIDC provider installed. To check for an OIDC provider first find the OIDC issuer with the following command. Substitute `$CLUSTER_NAME` with the *Name* of the EKS cluster:
+
```

Comment:
s/the/your/

Reply:
Fixed in [33bcbb5](https://github.com/SUSE-Enceladus/marketplace-docs/pull/14/commits/33bcbb5a2051fdfe1f229cfb0daa50c5c1423086).